docs/index.rst

```diff
@@ -30,7 +29,6 @@ Contents:
    :maxdepth: 2
 
    shotgun_model
-   shotgun_hierarchy_model
```

Comment:
Correct. The python-api method is also not official yet, so holding off for now.